src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,61 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private final char separator = ',';
```

Comment:
```suggestion
    private static final String COMA_SEPARATOR = ",";
```
avoid using `String.valueOf()`